Analysis/src/AnalysisCategories.cpp

```diff
@@ -25,6 +25,21 @@ const boost::bimap<std::string, analysis::EventRegion>& EventRegionMapToString()
 
 namespace analysis {
 
+boost::optional<EventRegion> EventRegion::_OS_Isolated;
+boost::optional<EventRegion> EventRegion::_OS_AntiIsolated;
+boost::optional<EventRegion> EventRegion::_SS_Isolated;
+boost::optional<EventRegion> EventRegion::_SS_LooseIsolated;
+boost::optional<EventRegion> EventRegion::_SS_AntiIsolated;
+
+void EventRegion::Initialize(DiscriminatorWP iso_lower, DiscriminatorWP anti_iso_lower, DiscriminatorWP anti_iso_upper)
+{
+    _OS_Isolated = EventRegion().SetCharge(true).SetLowerIso(iso_lower);
+    _OS_AntiIsolated = EventRegion().SetCharge(true).SetLowerIso(anti_iso_lower);
+    _SS_Isolated = EventRegion().SetCharge(false).SetLowerIso(iso_lower);
+    _SS_LooseIsolated = EventRegion().SetCharge(false).SetLowerIso(iso_lower);
```

Comment:
for the moment for `_SS_LooseIsolated` keep `Loose` in the code, instead `iso_lower`. We'll probably remove this EventRegion in the near future.